Systems/electrical.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<PropertyList>
+
+    <filter>
+        <type>gain</type>
+        <gain>100.0</gain>
+        <input>
+            <property>/systems/electrical/battery-charge-percent</property> <!-- this property ranges from 0.0 to 1.0 despite its name -->
+        </input>
+        <reference>0.0</reference>
```

Comment:
Oops, I will remove it. I used another filter as template and I just changed the values to output the correct thing without thinking if the statements were necessary. Sorry!